orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -132,7 +132,7 @@
 						infModifiedResidueInst.setAttributeValue(_displayName, modifiedResidueInst.getAttributeValue(_displayName));
 						// Update name to reflect that coordinate values are taken from humans. This takes place after cache retrieval, since the name from DB won't contain updated name.
 						if (modifiedResidueInst.getAttributeValue(coordinate) != null) {
-							String newModifiedResidueDisplayName = modifiedResidueInst.getAttributeValue(_displayName).toString() + " (in Homo sapiens)";
+							String newModifiedResidueDisplayName = modifiedResidueInst.getAttributeValue(_displayName).toString() + " (in reference species)";
```

Comment:
This will need to be replaced with a variable containing the reference species full name (I can add this if you want @preecej )

Reply:
Go for it. I considered it, but was moving  fast.

Reply:
:racing_car:

Reply:
done